controllers/volumereplicationgroup_controller.go

```diff
@@ -322,6 +327,19 @@ func (v *VRGInstance) processVRG() (ctrl.Result, error) {
 		return ctrl.Result{Requeue: true}, nil
 	}
 
+	if err := v.updateReplicationClassList(); err != nil {
```

Comment:
(optimize) can this be postponed till we need it the first time? IOW, make the call when we need it, so that we are done with other checks like ready for primary/secondary etc.?